README.md

```diff
@@ -65,20 +100,20 @@ module.exports = {
 
 With source maps enabled, you may also need to configure the extension settings/launch.json config to add customized paths between your runtime urls and your workspace paths, see [Sourcemaps](#sourcemaps) for more information.
 
+#### Debug Configuration
 
-### Debug Configuration
-You can launch the Microsoft Edge Tools extension like you would a debugger, by using a `launch.json` config file. However, Microsoft Edge Tools isn't a debugger and so any breakpoints set in VS Code won't be hit, you can of course use a different debug extension instead and attach the Microsoft Edge Tools extension once debugging has started.
+You can launch the Microsoft Edge Tools extension like you would a debugger, by using a `launch.json` config file. However, Microsoft Edge Tools isn't a debugger and so any breakpoints set in Visual Studio Code won't be hit, you can use a different debug extension instead and attach the Microsoft Edge Tools extension once debugging has started.
```

Comment:
my 2c:
`...like you would a debugger, by using a `launch.json` config file` 
this is not totally clear to me :), maybe ellaborating a little more on this?

`However, Microsoft Edge Tools isn't a debugger and so any breakpoints set in Visual Studio Code won't be hit, you can use a different debug extension instead and attach the Microsoft Edge Tools extension once debugging has started.`
what about: 
`Microsoft Edge Tools works great when paired with [Debugger for Microsoft Edge](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-edge), you can use the first one to design your frontend and the latter to debug your code and set breakpoints`

Reply:
Great re-write, done. Let's do the explanation of the launch.json file next, but I really want this to get out today as the legal names for VS Code need to be in there.